barbados/BCO/isotopes.yaml

```
@@ -0,0 +1,23 @@
+plugins:
+  source:
+    - module: intake_xarray
+
+sources:
+  precipitation:
+    description: Calibrated stable water isotope data in precipitation from the BCO during EUREC4A
+    driver: netcdf
+    args:
+      urlpath: simplecache::https://observations.ipsl.fr/aeris/eurec4a-data/BARBADOS/BCO/precipitation/EUREC4A_BCO_stable_water_isotopes_precipitation.nc
+      chunks: null
+    metadata:
+      maintainer: Leonie Villiger <leonie.villiger[at]env.ethz.ch>, Franziska Aemisegger <franziska.aemisegger[at]env.ethz.ch>
+      doi: https://doi.org/10.25326/242
```

We measured isotopes in water vapour with two instruments (LGR and Picarro) in parallel. Thus, the second dataset of isotopes in water vapour is missing. 
```suggestion
  water_vapor:
  description: BCO 
  driver: netcdf
  args:
  urlpath: simplecache::https://observations.ipsl.fr/aeris/eurec4a-data/BARBADOS/BCO/PICARRO/EUREC4A_BCO_PICARRO_WVISOS_L1.nc
  chunks: null
  metadata:
  mainainer: Franziska Aemisegger <franziska.aemisegger[at]env.ethz.ch>, Leonie Villiger <leonie.villiger[at]env.ethz.ch>
  doi: https://doi.org/10.25326/245
```

---

Thanks @leonie-villiger that is very helpful to know. How can these two water vapor datasets best be distinguished? What would be a meaningful and general distinct access pattern?
- `cat.BCO.isotopes.water_vapor.CRDS` and `cat.BCO.isotopes.water_vapor.OA_ICOS` (name from Bailey et al.)
- `cat.BCO.isotopes.water_vapor.PICARRO` and `cat.BCO.isotopes.water_vapor.LGR`
- `cat.BCO.isotopes.water_vapor.dry_mole_fraction` and `cat.BCO.isotopes.water_vapor.wet_mole_fraction`

Thinking about the other water vapor datasets on Meteor, ATR, P-3, it seems to me that the type of mole fraction might be the most general identifier that would also work for those datasets. What do you think?

---

I would stick with the naming from Bailey et al., i.e., `*.CRDS` and `.OA_ICOS`, since it is the official EUREC4A paper describing all the isotope datat set.